app/models/placement_group.rb

```diff
@@ -0,0 +1,81 @@
+class PlacementGroup < ApplicationRecord
+  acts_as_miq_taggable
+  include SupportsFeatureMixin
+  include NewWithTypeStiMixin
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  has_many :vms, :dependent => :nullify
+
+  delegate :queue_name_for_ems_operations, :to => :ext_management_system, :allow_nil => true
+
+  def self.class_by_ems(ext_management_system)
+    ext_management_system&.class_by_ems(:PlacementGroup)
+  end
+
+  def self.my_zone(ems)
+    # TODO(pblaho): find unified way how to do that
+    ems ? ems.my_zone : MiqServer.my_zone
+  end
+
+  def my_zone
+    self.class.my_zone(ext_management_system)
+  end
+
+  def self.create_placement_group_queue(userid, ext_management_system, options = {})
+    task_opts = {
+      :action => "creating Placement Group for user #{userid}",
+      :userid => userid
+    }
+
+    queue_opts = {
+      :class_name  => 'ManageIQ::Providers::CloudManager::PlacementGroup',
+      :method_name => 'create_placement_group',
+      :role        => 'ems_operations',
+      :queue_name  => ext_management_system.queue_name_for_ems_operations,
+      :zone        => ext_management_system.my_zone,
+      :args        => [ext_management_system.id, options]
+    }
+
+    MiqTask.generic_action_with_callback(task_opts, queue_opts)
+  end
+
+  def self.create_placement_group(ems_id, options = {})
+    raise ArgumentError, _("ems_id cannot be nil") if ems_id.nil?
+
+    ext_management_system = ExtManagementSystem.find(ems_id)
+    raise ArgumentError, _("ext_management_system cannot be found") if ext_management_system.nil?
+
+    klass = ext_management_system.class_by_ems(:Placement)
+    klass.raw_create_placement_group(ext_management_system, options)
+  end
+
+  def self.raw_create_placement_group(_ext_management_system, _options = {})
+    raise NotImplementedError, _("raw_create_placement_group must be implemented in a subclass")
+  end
+
+  def delete_placement_group_queue(userid)
+    task_opts = {
+      :action => "deleting Placement Group for user #{userid}",
+      :userid => userid
+    }
+
+    queue_opts = {
+      :class_name  => self.class.name,
+      :method_name => 'delete_placement_group',
+      :instance_id => id,
+      :role        => 'ems_operations',
+      :queue_name  => ext_management_system.queue_name_for_ems_operations,
+      :zone        => ext_management_system.my_zone,
```

Comment:
You're already delegating `queue_name_for_ems_operations` here https://github.com/ManageIQ/manageiq/pull/21807/files#diff-2e7e2b5552f1a0d66eff90857b08fa0fc1ca3d26c00cdc9f7b0f8d6932300b37R8 and have `my_zone` defined here https://github.com/ManageIQ/manageiq/pull/21807/files#diff-2e7e2b5552f1a0d66eff90857b08fa0fc1ca3d26c00cdc9f7b0f8d6932300b37R19-R21
```suggestion
      :queue_name  => queue_name_for_ems_operations,
      :zone        => my_zone,
```